Sources/MapboxMaps/Gestures/GestureOptions.swift

```diff
@@ -73,5 +73,13 @@ public struct GestureOptions: Equatable {
     /// Defaults to `UIScrollView.DecelerationRate.normal.rawValue`
     public var panDecelerationFactor: CGFloat = UIScrollView.DecelerationRate.normal.rawValue
 
+    /// Whether animations (existing and new) are automatically canceled while a gesture is
+    /// in progress. Cancellation begins after a short delay to allow short, discrete gestures
+    /// like double tap and double touch to be used repeatedly without interruption. Set this
+    /// flag to `false` in situations where a gestures should be allowed to run in parallel
+    /// with animations (e.g. when an animation is modifying the camera center and a
+    /// gesture should be allowed to modify zoom). Defaults to `true`.
+    public var disableAnimationsDuringGestures: Bool = true
```

Comment:
here's the new public gesture option

Reply:
do you have an example on how to use this option together with the viewport to customise gestures?

Reply:
Also do you feel this flag redundant with the `transitionToIdleUponUserInteraction` flag in the viewport? e.g `transitionToIdleUponUserInteraction = false` must go with `disableAnimationsDuringGestures = false`?

Reply:
after discussion: we'll remove this option and control the behavior using the viewport option

Reply:
Updated